src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1448,6 +1543,11 @@ type Money {
     amount: Float!
 }
 
+""""
+An ISO-8601-formatted datetime, i.e. `2023-06-13T14:30` for 2:30pm on June 13th 2023.
```

Comment:
```suggestion
An ISO-8601-formatted datetime with offset, i.e. `2023-06-13T14:30+03:00` for 2:30pm on June 13th 2023 at Helsinki's offset from UTC at that time.
```

Reply:
We might want to call the scalar `OffsetDateTime` to be super-clear (but we can discuss that).

If we ever need a `LocalDateTime` we can also add that later.

Reply:
Yes I thought too it makes sense to have the offset in the scalar name so at some point we have the datetime name available later on if we need a more generic datetime and the offset in name does make it more obvious that this is a limited part of the datetime spec.